packages/quantic/force-app/main/default/lwc/quanticSearchBoxSuggestionsList/quanticSearchBoxSuggestionsList.js

```diff
@@ -0,0 +1,64 @@
+import {
+  api,
+  LightningElement,
+  track
+} from 'lwc';
+
+export default class QuanticSearchBoxSuggestionsList extends LightningElement {
+  /** @type {{key: number, value: string, rawValue: string}[]} */
+  @api suggestions = [];
+  /** @type {number} */
+  @track selectionIndex = -1;
+
+  renderedCallback() {
+    if(this.selectionIndex > -1) {
+      this.emitSuggestionHighlighted();
+    }
+  }
+
+  @api selectionUp() {
```

Comment:
Oh we're getting fancy with the decorators on the side now are we?
Jk i have no preference except I think they should be the same everywhere

Reply:
Whats the rule?